doc/man3/RAND_add.pod

```diff
@@ -78,6 +69,72 @@ RAND_event() returns RAND_status().
 
 The other functions do not return values.
 
+=head1 NOTES
+
+The new OpenSSL DRBG has some peculiarities which need to be taken
+into account when it is selected as the default OpenSSL CSPRNG, i.e.,
+when RAND_get_rand_method() == RAND_OpenSSL().
+This applies in particular to the way reseeding is done by the DRBG:
+
+=over 2
+
+=item *
+
+The DRBG seeds itself automatically, pulling random input from trusted
+entropy sources.
+Automatic reseeding occurs after 2^24 generate requests.
```

Comment:
I think we should avoid documenting implementation details

Reply:
I assume that the only thing that disturbs you here is the fact that I mention a specific number here? So I changed the formulation as follows:

> Automatic reseeding occurs after a predefined number of generate requests.

Reply:
Works for me